ext/build_defaults.yaml

```diff
@@ -16,3 +16,4 @@ apt_signing_server: 'apt.puppetlabs.com'
 apt_repo_url: 'http://apt.puppetlabs.com'
 apt_repo_path: '/opt/repository/incoming'
 tar_host: 'downloads.puppetlabs.com'
+nonfinal_gem_path: '/opt/repository-nightlies/downloads/gems/puppet5'
```

Comment:
This should probably be `puppet6-nightly` (and the 5.5.x branch should be `puppet5-nightly`)

Reply:
@mwaggett there is no `puppet5-nightly` at http://nightlies.puppet.com/downloads/gems/index_by_name.html. Should the 5.5.x branch still be changed?

Reply:
@melissa I think they were originally shipped under `puppet5` and then removed so we could use the `-nightly` repo names. @gimmyxd should be able to confirm these paths.

Reply:
puppet-agent pipeline is failing due to rubygems issues so the ship job never run. Also the job will still fail until the next release of packaging, so thats why there are no puppet5-nightly gems